scalafix-tests/src/test/resources/checkSyntax/NoXml.source

```diff
@@ -0,0 +1,99 @@
+rewrites = [NoXml]
+
+<<< don't add import when not needed
+object A {}
+>>>
+object A {}
+<<< single line, no splice
+object A {
+  <div></div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  xml"""<div></div>"""
+}
+
+<<< single line, splice
+object A {
+  val bar = "bar"
+  <div>{bar}</div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  val bar = "bar"
+  xml"""<div>${bar}</div>"""
+}
+
+<<< multi-line, no splice
+object A {
+  val foo =
+    <div>
+      <span>Hello</span>
+    </div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  val foo =
+    xml"""<div>
+      <span>Hello</span>
+    </div>"""
+}
+
+<<< multi-line, splice
+object A {
+  val foo =
+    <div>
+      <span>{"Hello"}</span>
+    </div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  val foo =
+    xml"""<div>
+      <span>${"Hello"}</span>
+    </div>"""
+}
+
+<<< splice in attribute position
+object A {
+  <foo bar={"Hello"}/>
+}
+>>>
+import scala.xml.quote._
+object A {
+  xml"""<foo bar=${"Hello"}/>"""
```

Comment:
Skip for now, this rewrite is actually syntactic so we shouldn't even need to typecheck with scalahost-nsc. It will be possible to run this rewrite from the cli without sbt-scalafix.